### DISCUSSION TITLE
Automatically load a Nuget Package

### DISCUSSION THREAD

**Discussion by fforay** [dotnet/interactive#3555] [Q&A]
I've written a new Language Kernel to support the XSharp Language (www.xsharp.eu) in order to provide some learning content for new comers.

Currently, in the beginning of each Notebook, I have a coding cell that contains the loading of the XSharpInteractive Nuget package :
#r "nuget:XSharpInteractive,1.0.0"

It would be great if we can have a setting (or something) to do automatically.

Or maybe it already exists and I've missed it ?

**Comment by jonsequitur** [comment]
There's been some discussion of having a "configuration" file that if it exists in the same directory as a notebook, it would be run automatically when the kernel is started. (This file would be a .dib which would be run using `#!import` so code in various languages could be included.)

**Reply by fforay** [reply]
That would be great !
